lib/Doctrine/ORM/EntityManager.php

```diff
@@ -191,6 +200,16 @@ public function getMetadataFactory()
     }
 
     /**
+     * Gets the persister factory used to manipulate collection and entity classes.
+     *
+     * @return \Doctrine\ORM\Persisters\PersisterFactory
+     */
+    public function getPersisterFactory()
```

Comment:
This method doesn't seem to fit the EntityManager at all: it would honestly be better to not expose it to end-users like this.